WordPress/Classes/Users/Views/UserDetailsView.swift

```diff
@@ -61,29 +63,34 @@ struct UserDetailsView: View {
                 }
             }
 
-            if !applicationTokenListViewModel.applicationTokens.isEmpty {
-                Section(ApplicationTokenListView.title) {
-                    ForEach(applicationTokenListViewModel.applicationTokens) { token in
-                        ApplicationTokenListItemView(item: token)
-                    }
-                }
-            }
-
-            if viewModel.currentUserCanModifyUsers {
+            if isCurrentUser || viewModel.currentUserCanModifyUsers {
                 Section(Strings.accountManagementSectionTitle) {
-                    Button(Strings.setNewPasswordActionTitle) {
-                        presentPasswordAlert = true
+                    if isCurrentUser {
```

Comment:
I think this is why I couldn't see Application Passwords for other users?

Can we make that possible? 

Doesn't have to be in this PR

Reply:
Oh sorry, I didn't realize [your question](https://github.com/wordpress-mobile/WordPress-iOS/pull/23834#issuecomment-2498879393) was about the current implementation in the app.

That's in my TODO list. It's not available now because the "Application Passwords" entry was implemented as for current user only.